phoenicis-tools/src/main/java/org/phoenicis/tools/system/opener/Opener.java

```diff
@@ -0,0 +1,23 @@
+package org.phoenicis.tools.system.opener;
+
+import java.net.URI;
+
+/**
+ * System Opener
+ */
+@FunctionalInterface
+public interface Opener {
+    /**
+     * Open a file
+     * @param file to be opened
+     */
+    void open(String file);
```

Comment:
Don't like "file" (it's also used e.g. for URLs). Doesn't it allow to inject code as well?